src/platform/intel/cavs/include/cavs/lib/mailbox.h

```diff
@@ -65,7 +65,7 @@
 #define MAILBOX_DSPBOX_BASE	SRAM_OUTBOX_BASE
 
 #define MAILBOX_SW_REG_SIZE	SRAM_SW_REG_SIZE
-#define MAILBOX_SW_REG_BASE	SRAM_SW_REG_BASE
+#define MAILBOX_SW_REG_BASE	((uintptr_t)SRAM_SW_REG_BASE)
```

Comment:
That looks inconsistent. A comment explaining why?

Reply:
@marc-hb Agree, that it looks a bit strange, and you know what - you're right, I'll solve this differently by moving these casts where `SRAM_SW_REG_BASE` and other "first generation" macros are defined, I'll drop this patch from this PR. Thanks for the heads-up!